app/models/preserved_copy.rb

```diff
@@ -16,8 +15,7 @@ class PreservedCopy < ApplicationRecord
     INVALID_MOAB_STATUS => 1,
     INVALID_CHECKSUM_STATUS => 2,
     ONLINE_MOAB_NOT_FOUND_STATUS => 3,
-    EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS => 4,
-    FIXITY_CHECK_FAILED_STATUS => 5,
+    UNEXPECTED_VERSION_ON_STORAGE_STATUS => 4,
     VALIDITY_UNKNOWN_STATUS => 6
```

Comment:
so we'll just skip 5  for now?    Ok with me.  Should there be a comment about it?

Reply:
yeah, i'd be inclined to skip 5 permanently.  because we have things that use 6.  since nothing used the removed status, which previously mapped to 5, we could conceivably re-use that code.  but i don't think we'll have tones of new statuses, so my preference would be to just keep using the next highest enum value as we add new statuses, without re-using old ones.  just seems lower effort.

there is already the comment about not changing the underlying constants, but probably best to add a clarification about why there might be gaps in the sequence?

Reply:
@ndushay, how about something like:

```ruby
# NOTE:  DO NOT change the underlying constants for enum values that have been merged to
# master/used in prod db (or at least, consider the necessary migration).
# If an enum value is removed, it's probably easiest to just not re-use the underlying int code it used before.
# Just add new statuses using the next highest unused int value, and treat the numbering as an increasing
# sequence that will likely have holes.
```

Reply:
fixed